apps/srp.c

```diff
@@ -553,6 +553,8 @@ int srp_main(int argc, char **argv)
             user = *(argv++);
         else {
```

Comment:
> you want a commit to add the braces to the if I can do that.

Yes, please :-) For reference, coding style guide is at https://www.openssl.org/policies/codingstyle.html.